### PR TITLE
zh: add tidbcluster configuration document

### PR DESCRIPTION
**Pull request by Yisaer**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)

Add tidbcluster configuration document

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/configure-tidb-cluster.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: 通过 TidbCluster 配置 TiDB 集群
+category: how-to
+---
+
+## TiDB 集群配置
+
+TidbCluster 文件支持在其上面直接配置 TiDB/TiKV/PD 的配置选项，本篇文章将介绍如何在 TiDBCluster 上配置参数。目前 Operator 1.1 版本支持了 TiDB 3.1 版本参数。针对各组件配置参数，请参考 PingCAP 官方文档。
```
Comment (DanielZhangQD) [inline zh/configure-tidb-cluster.md]: Pump

Comment (DanielZhangQD) [inline zh/configure-tidb-cluster.md]:
```suggestion
TidbCluster 文件支持在其上面直接配置 TiDB/TiKV/PD 的配置选项，本篇文章将介绍如何在 TiDBCluster 上配置参数。目前 Operator 1.1 版本支持了 TiDB 集群 v3.1 版本参数。针对各组件配置参数，请参考 PingCAP 官方文档。
```

File: zh/configure-tidb-cluster.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+## 配置 TiDB 配置参数
+
+你可以通过 `TidbCluster.Spec.Tidb.Config` 来配置 TiDB 配置参数，以下是一个例子。获取所有可以配置的 TiDB 配置参数，请参考 [TiDB 配置文档](https://pingcap.com/docs-cn/v3.1/reference/configuration/tidb-server/configuration-file/)
+
+> **注意：**
+>
+> 为了兼容 `helm` 部署，如果你是通过 CR 文件部署 TiDB 集群，即使你不设置 Config 配置，也需要保证 `Config: {}` 的设置，从而避免 TiDB 组件无法正常启动。
```
Comment (DanielZhangQD) [inline zh/configure-tidb-cluster.md]: `Config` is a pointer, why do we need this in CR if no config required?

File: zh/configure-tidb-cluster.md (hunk 1 above)
Comment (DanielZhangQD) [reply zh/configure-tidb-cluster.md]: An example I ever tested:
```
  pump:
    baseImage: pingcap/tidb-binlog
    version: v3.0.11
    replicas: 1
    storageClassName: local-storage
    schedulerName: default-scheduler
    config:
      addr: 0.0.0.0:8250
      gc: 7
      data-dir: /data
      heartbeat-interval: 2
      pd-urls: http://dan10-pd:2379
```

File: zh/configure-tidb-cluster.md (hunk 2 above)
Comment (Yisaer) [reply zh/configure-tidb-cluster.md]: To be compatible with helm.

File: zh/configure-tidb-cluster.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
```
Comment (DanielZhangQD) [inline zh/configure-tidb-cluster.md]: Please update TOC.md

File: zh/configure-tidb-cluster.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+title: 通过 TidbCluster 配置 TiDB 集群
+category: how-to
```
Comment (ran-huang) [inline zh/configure-tidb-cluster.md]: Please add a one-sentence summary under `title`.
```suggestion
title: 通过 TidbCluster 配置 TiDB 集群
summary: 
category: how-to
```

File: zh/configure-tidb-cluster.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+---
+
+## TiDB 集群配置
+
+TidbCluster 文件支持在其上面直接配置 TiDB/TiKV/PD 的配置选项，本篇文章将介绍如何在 TiDBCluster 上配置参数。目前 Operator 1.1 版本支持了 TiDB 集群 v3.1 版本参数。针对各组件配置参数，请参考 PingCAP 官方文档。
+
+
```
Comment (ran-huang) [inline zh/configure-tidb-cluster.md]:
```suggestion

```

File: zh/configure-tidb-cluster.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+## 配置 TiDB 配置参数
+
+你可以通过 `TidbCluster.Spec.Tidb.Config` 来配置 TiDB 配置参数，以下是一个例子。获取所有可以配置的 TiDB 配置参数，请参考 [TiDB 配置文档](https://pingcap.com/docs-cn/v3.1/reference/configuration/tidb-server/configuration-file/)
+
+> **注意：**
+>
+> 为了兼容 `helm` 部署，如果你是通过 CR 文件部署 TiDB 集群，即使你不设置 Config 配置，也需要保证 `Config: {}` 的设置，从而避免 TiDB 组件无法正常启动。
+
```
Comment (ran-huang) [inline zh/configure-tidb-cluster.md]:
```suggestion
```

File: zh/configure-tidb-cluster.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+---
+
+## TiDB 集群配置
+
+TidbCluster 文件支持在其上面直接配置 TiDB/TiKV/PD 的配置选项，本篇文章将介绍如何在 TiDBCluster 上配置参数。目前 Operator 1.1 版本支持了 TiDB 集群 v3.1 版本参数。针对各组件配置参数，请参考 PingCAP 官方文档。
+
+
+## 配置 TiDB 配置参数
+
+你可以通过 `TidbCluster.Spec.Tidb.Config` 来配置 TiDB 配置参数，以下是一个例子。获取所有可以配置的 TiDB 配置参数，请参考 [TiDB 配置文档](https://pingcap.com/docs-cn/v3.1/reference/configuration/tidb-server/configuration-file/)
```
Comment (ran-huang) [inline zh/configure-tidb-cluster.md]: Please also add a period in TiKV and PD section.
```suggestion
你可以通过 `TidbCluster.Spec.Tidb.Config` 来配置 TiDB 配置参数，以下是一个例子。获取所有可以配置的 TiDB 配置参数，请参考 [TiDB 配置文档](https://pingcap.com/docs-cn/v3.1/reference/configuration/tidb-server/configuration-file/)。
```

File: zh/configure-tidb-cluster.md (hunk 3 above)
Comment (Yisaer) [reply zh/configure-tidb-cluster.md]: updated.

**Comment by Yisaer** [comment]
PTAL @DanielZhangQD @ran-huang

**Comment by ran-huang** [comment]
I think the file name of `configure-tidb-cluster.md` looks very like [`configure-a-tidb-cluster.md`](https://github.com/pingcap/docs-tidb-operator/blob/master/zh/configure-a-tidb-cluster.md).
We should consider changing it to `configure-cluster-using-tidbcluster.md`.

**Comment by Yisaer** [comment]
> I think the file name of `configure-tidb-cluster.md` looks very like [`configure-a-tidb-cluster.md`](https://github.com/pingcap/docs-tidb-operator/blob/master/zh/configure-a-tidb-cluster.md).
> We should consider changing it to `configure-cluster-using-tidbcluster.md`.

updated.

File: zh/configure-tidb-cluster.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
+---
+title: 通过 TidbCluster 配置 TiDB 集群
+summary:
```
Comment (ran-huang) [inline zh/configure-tidb-cluster.md]:
```suggestion
summary: 介绍如何使用 TidbCluster 配置 TiDB 集群。
```

File: zh/configure-tidb-cluster.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
+---
+title: 通过 TidbCluster 配置 TiDB 集群
+summary: 
+category: how-to
+---
+
+## TiDB 集群配置
```
Comment (ran-huang) [inline zh/configure-tidb-cluster.md]: Please keep the top level heading consistent with the title in Line 2.
```suggestion
# 通过 TidbCluster 配置 TiDB 集群
```

File: zh/configure-tidb-cluster.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+TidbCluster 文件支持在其上面直接配置 TiDB/TiKV/PD 的配置选项，本篇文章将介绍如何在 TiDBCluster 上配置参数。目前 Operator 1.1 版本支持了 TiDB 集群 v3.1 版本参数。针对各组件配置参数，请参考 PingCAP 官方文档。
+## 配置 TiDB 配置参数
```
Comment (ran-huang) [inline zh/configure-tidb-cluster.md]: Please add a blank line both before and after every heading.
```suggestion

## 配置 TiDB 配置参数
```

**Comment by ran-huang** [comment]
> > I think the file name of `configure-tidb-cluster.md` looks very like [`configure-a-tidb-cluster.md`](https://github.com/pingcap/docs-tidb-operator/blob/master/zh/configure-a-tidb-cluster.md).
> > We should consider changing it to `configure-cluster-using-tidbcluster.md`.
> 
> updated.

Please change the name of the `configure-tidb-cluster.md` file to `configure-cluster-using-tidbcluster.md` and update the link in TOC.

**Comment by Yisaer** [comment]
> change the name of the `configure-tidb-cluster.md` file to `configure-cluster-using-tidbcluster.md` and update the link in TOC.

It seems already updated. Maybe you need to refresh the change.

**Comment by ran-huang** [comment]
LGTM.
@DanielZhangQD PTAL~

**Pull request merged**